hack/ingress-nginx.yaml

```diff
@@ -0,0 +1,687 @@
+
```

Comment:
As @rootfs has asked before, why do you need the ingress?

And in the case it is necessary, this yaml should be in the `manifests/kind` folder, not in the `hack`...

Reply:
removed